v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner.md

```diff
@@ -295,7 +296,16 @@ expoert WORKER_REGION="${REGION}"
 export NETWORK="<VPC_NAME>"
 #### Subnet where you would like to run Dataflow. Defaults to default. This subnet must have access to Cassandra nodes you would like to migrate from.
 export SUBNETWORK="regions/${WORKER_REGION}/subnetworks/<SUBNET_NAME>"
-
+#### Number of partitions for parallel read.
+##### By default Apache Beam's CassandraIO sets NUM_PARTITIONS equals to number
+##### of nodes on the Cassandra Cluster. This default does not give good performance
+##### larger workloads as it limits the parallelization.
+##### While specifcs would depend on many factors like number of Cassandra nodes, distribution
+##### of partitions of the table across the nodes,
```

Comment:
Should we mention explicitly that for non cassandra this need not be tuned?

Reply:
This is in a section dedicated for Cassandra.